test/jasmine/tests/axes_test.js

```diff
@@ -6416,6 +6416,707 @@ describe('Test axes', function() {
             });
         });
     });
+
+    describe('minor ticks"', function() {
+        var gd;
+
+        beforeEach(function() {
+            gd = createGraphDiv();
+        });
+
+        afterEach(destroyGraphDiv);
+
+        function _assert(expPositions) {
+            var ax = gd._fullLayout.xaxis;
+
+            // minor positions
+            var positions =
+                ax._vals
+                    .filter(function(d) { return d.minor; })
+                    .map(function(d) { return d.x; });
```

Comment:
nonblocking: the date and log tests would be more readable if the values were in data format, ie I think `ax.l2d(d.x)`